system_api/cypher.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    api_client = ApiClient(configuration=Configuration(server_index=2),header_name="x-api-key", header_value=API_KEY)
</pre><p>Should we deploy to prod so that this can hit the prod endpoint? So customers can use their api key with the same cookbook and view results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SystemInternal/SystemCookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>